docusaurus/docs/what-is-included/top-progress-bar.md

```diff
@@ -0,0 +1,21 @@
+---
+id: top-progress-bar
+title: Top progress bar
+sidebar_label: Top progress bar
+---
+
+A very subtle progress bar is displayed when Next.js is performing route changes, which can take some time depending on how fast the page assets and data (via `getInitialProps`, `getServerSideProps` or `getStaticProps` with `revalidate`).
```

Comment:
Small detail, this sentence seems to be incomplete 🤔 
```suggestion
A very subtle progress bar is displayed when Next.js is performing route changes, which can take some time depending on how fast the page assets and data are loaded (via `getInitialProps`, `getServerSideProps` or `getStaticProps` with `revalidate`).
```

Not sure this is the clearest wording to use, but just to exemplify what seems to be missing.